otel-collector-config.yml

```diff
@@ -0,0 +1,41 @@
+receivers:
+  otlp:
```

Comment:
it is nice to include the otlp receiver. so on local, you can push other traces in using `telemetrygen` with a command like this:
`$GOBIN/telemetrygen traces --otlp-insecure --service my-service --traces 3`

https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/cmd/telemetrygen